docs/dev_docs/dev_notes.rst

```diff
@@ -101,6 +101,7 @@ For a quick test of environment import ``uno`` If there is no import  error you
 
 .. code-block:: text
 
+    PS C:\python_ooo_dev_tools> .env\scripts\activate
```

Comment:
This is not good enough. Linux and Windows section venv functionality needs to be the same.